packages/orbit-tailwind-preset/src/presets/components/index.ts

```diff
@@ -76,7 +76,7 @@ const getBackgroundColors = (tokens: typeof defaultTokens) => {
 };
 
 const cfg = (options?: Options): Config => {
-  const { disablePreflight = true } = options || {};
+  const { disablePreflight = false } = options || {};
```

Comment:
I ran this test in this PR's branch:

```js
> require("@kiwicom/orbit-tailwind-preset")().corePlugins.preflight
undefined
> require("@kiwicom/orbit-tailwind-preset")({disablePreflight: false}).corePlugins.preflight
undefined
> require("@kiwicom/orbit-tailwind-preset")({disablePreflight: true}).corePlugins.preflight
false
```

I assume Tailwind injects the preflight by default so `undefined` means preflight enabled? Is this correct?

We can also be more explicit on line 87 and always set a boolean:

```js
corePlugins: {
  preflight: !disablePreflight,
},
```

What do you think? This way we don't need to know that preflight is true by default.

Reply:
agree, updated